src/features/Apiexplorer/RequestResponseRenderer/index.tsx

```diff
@@ -21,6 +22,15 @@ type TPlaygroundSection<T extends TSocketEndpointNames> = {
 };
 
 export const LoginModal = (visible) => {
+  const { getUrl } = useLoginUrl();
+
+  const handleClick = () => {
```

Comment:
Please provide test cases for these as well. thanks :pray:

Reply:
Hubert is writing tests for the component so it can  be done in that

Reply:
oh nice, on the card, right. thanks man :pray: